src/melee.cpp

```diff
@@ -603,8 +603,10 @@ void Character::melee_attack( Creature &t, bool allow_special, const matec_id &f
     const int encumbrance_cost = roll_remainder( ( encumb( bp_arm_l ) + encumb( bp_arm_r ) ) *
                                  2.0f );
     const int deft_bonus = hit_spread < 0 && has_trait( trait_DEFT ) ? 50 : 0;
+    float skill_cost = ( 30 - melee ) / 30;
+    skill_cost = std::max( 0.667f, skill_cost );
```

Comment:
```suggestion
    const float skill_cost = std::max( 0.667f, ( 30 - melee ) / 30 );
```
this line could be furthur simplified.